src/main.c

```diff
@@ -501,6 +527,8 @@ int main(int argc, const char **argv) {
 		}
 	} else if (!strcmp (argv[db0 + 1], "=")) {
```

Comment:
what about checking char by char instead of two consecurive strcmps?

Reply:
Instead of
```c
	} else if (*argv[db0 + 1] == '=') {
		ret = createdb (argv[db0], NULL, 0);
	} else if (!strcmp (argv[db0 + 1], "==")) {
		ret = createdb (argv[db0], argv + db0 + 2, argc - (db0 + 2));
	} else {
		s = sdb_new (NULL, argv[db0], 0);
		if (!s) {
			return 1;
		}
		...
```
you would then have something like:
```c
	} else if (*argv[db0 + 1] == '=') {
		if (argv[db0 + 1][1] == '\0') {
			ret = createdb (argv[db0], NULL, 0);
		} else if (argv[db0 + 1][1] == '=' && argv[db0 + 1][2] == '\0') {
			ret = createdb (argv[db0], argv + db0 + 2, argc - (db0 + 2));
		} else {
			goto els;
		}
	} else {
els:
		s = sdb_new (NULL, argv[db0], 0);
		if (!s) {
			return 1;
		}
		...
```
So I think the first is better.

Reply:
No. This is ultracomplicated. You can nest one condition under the other. And make it shorter but anyway im in phone and strcmp is fine